install/cupy_builder/cupy_setup_build.py

```diff
@@ -512,9 +503,9 @@ def check_extensions(extensions):
             if not path.isfile(f):
                 raise RuntimeError('''\
 Missing file: {}
-Please install Cython {} or later. Please also check the version of Cython.
+Please install Cython.
 See https://docs.cupy.dev/en/stable/install.html for details.
```

Comment:
From the docs: 
Cython 0.29.22 or later is required to build CuPy from source. It will be automatically installed during the build process if not available.


Here it doesn't specify that Cython should be less than 3.0
there is no upper bound in Cython currently? I think we should modify the docs to specify this.